src/ontology/ro-edit.owl

```diff
@@ -6147,6 +6149,17 @@ AnnotationAssertion(rdfs:label obo:RO_0015003 "subcluster of")
 SubObjectPropertyOf(obo:RO_0015003 obo:BFO_0000050)
 TransitiveObjectProperty(obo:RO_0015003)
 
+# Object Property: obo:RO_0015004 (has characterizing marker set)
+
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0015004 "transcriptomically defined cell type X subClass_of: (has_characterizing_marker_set some S1); S1 has_part some gene 1, S1 has_part some gene 2, S1 has_part some gene 3, S1 has_anatomical_context some [region]")
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0015004 "A relation between a material entity and a set of markers that indentifies it in a particular context.")
+AnnotationAssertion(obo:IAO_0000116 obo:RO_0015004 "C has_characterizing_marker_set y iff x is a context in which C resides and y is a set of markers that is sufficient to identify C in context x.")
+AnnotationAssertion(dce:contributor obo:RO_0015004 <https://orcid.org/0000-0001-7258-9596>)
```

Comment:
Not sure we need something so formal, but if we keep this - suggest relaxing to

C has_characterizing_marker_set y iff: C is a cell type and y is a collection of genes or proteins whose expression is sufficient to distinguish cell type C from most or all other cell types.

Reply:
Also odd that the more formal bit is in editor's note.

It would also be good to have some guidance e.g. do not use this for cases where set of genes/proteins are only useful as markers in some specific context - e.g. in some specific location.  In these cases it is recommended to make a more specific cell class restricted to the relevant context.